.github/actions/create-deployment-package/action.yml

```diff
@@ -3,7 +3,15 @@ description: "Create a deployment package for Headless and Atomic"
 runs:
   using: composite
   steps:
-    - id: parse-version
+    - uses: actions/setup-python@v2
+      with:
+        python-version: '3.8'
```

Comment:
The example I was given uses python 3.8. I was told that Python 3.9 and 3.10 should work too, but since the only thing we run in python is the deployment package, I don't see a reason to upgrade (yet).